mlrun/api/api/endpoints/runs.py

```diff
@@ -147,6 +147,42 @@ async def get_run(
     }
 
 
+@router.get("/projects/{project}/completed_runs")
+async def get_completed_runs(
+    project: str,
+    since_datetime: str,
+    iter: int = 0,
```

Comment:
iter has a meaning when trying to get a single run. It marks 1 iteration of a "hyper run".
For list runs, iter is a boolean and when True it gets the the runs with iteration 0. Its not needed here, let remove it.

Reply:
Thanks for letting me know! I didn't realize that. 

I removed the parameter for this endpoint. Let me know if that's what we're expecting for it.